make.js

```diff
@@ -55,6 +57,10 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     for (var a = 0; a < apis.length; a++) {
         makeApiFiles(apis[a], sourceDir, apiOutputDir);
     }
+
+    var testXMLRefDocsJsonTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/XMLRefDocs.json.ejs"));
+    writeFile(path.resolve(apiOutputDir, "test/", "XMLRefDocs.json.autogen"), testXMLRefDocsJsonTemplate(locals));

```

Comment:
so is this generating to the SDK output? do we care?

also small thing, but would it make more sense to do this before makeApiFiles? if XMLRefDocs.json is consumed when generating the API files it seems like you should do that first

Reply:
this is making a .autogen file, but thinking about this more i can keep the checked in version blank as that's just overrides